spark-print-data/printertypes.json

```diff
@@ -662,6 +662,85 @@
 				"icons.100x100_id",
                 "build_volume.bed_file_id"
             ]
-        }
+        },
+        {
+            "id": "FA1118E1-7A59-FF4B-B938-D4D37788792F",
+            "version": 1,
+            "name": "Farm 1",
+            "manufacturer": "OCEAN",
+            "registration_url" : null,
+            "model_number": "1.0.0",
+            "icons": {
+                "16x16_id": "data/OCFarm1_16x16.png",
+                "32x32_id": "data/OCFarm1_32x32.png",
+                "50x50_id": "data/OCFarm1_50x50.png",
+                "100x100_id": "data/OCFarm1_100x100.png"
+            },
+            "technology": "FDM",
+            "default_material_id": "081011F3-4DE4-4580-9456-5F862DCAC3F0",
+            "default_profile_id": "F5641294-BD6B-44F9-8C34-1DDEB9394296",
```

Comment:
This profile_id does not exist please use one of the ones you created in the profiles sections